mr/src/test/java/org/apache/iceberg/mr/hive/TestTables.java

```diff
@@ -115,7 +115,7 @@ public String catalogName() {
   /**
    * The table properties string needed for the CREATE TABLE ... commands,
    * like "TBLPROPERTIES('iceberg.catalog'='mycatalog')
-   * @return
+   * @return The tables properties string, such as "TBLPROPERTIES('iceberg.catalog'='mycatalog')
```

Comment:
nit: same here

Reply:
I think quotes are not aligned too

Reply:
ah this is just a blind copy/paste from the javadoc description a few lines above without actually me checking whether it had missing pieces..will update

Reply:
fixed